packages/experimental/Spinner/src/Spinner.types.tsx

```diff
@@ -55,11 +55,20 @@ export interface SpinnerProps extends ViewProps, SpinnerTokens {
    * @defaultValue 'active'
    */
   status?: SpinnerStatus;
-  S;
   /**
    * Spinner label
    */
   label?: string;
+  /**
+   * Spinner animating or not
+   * @defaultValue 'true'
+   */
+  animating?: boolean;
```

Comment:
I am understanding these as mobile specific usages - do you also plan to update the documentations indicating the platform in this PR?

Reply:
Yes.

Reply:
Updated the platform , will update on spec once that is in from win32 side.